pkg/errutil/multierror.go

```diff
@@ -32,6 +32,11 @@ func (es MultiError) Err() error {
 	return NonNilMultiError(es)
 }
 
+// Error implements the error interface.
+func (es MultiError) Error() string {
```

Comment:
Let's check the recent error handling changes

https://github.com/thanos-io/thanos/pull/3833
https://github.com/thanos-io/thanos/pull/3836

@yashrsharma44

Reply:
Done, have used the new Error changes 👍